flink-runtime/src/test/java/org/apache/flink/runtime/operators/coordination/CoordinatorEventsExactlyOnceITCase.java

```diff
@@ -169,6 +171,9 @@ public static void shutdownMiniCluster() throws Exception {
 
     @Test
     public void test() throws Exception {
```

Comment:
nit: Maybe give a bit more descriptive name what this test does. E.g. `testExactlyOnceProcessingGuaranteesUnderTaskFailures`